docs/stdlib/array.rst

```diff
@@ -101,27 +100,25 @@ Reference
 
     :index: array
 
-    Arrays represent a one-dimensional homogeneous ordered list.
+    Represents a one-dimensional array of an homogeneous ordered list.
 
-    Array indexing starts at zero.
+    Array indexing will always start at zero. With the exception of other
```

Comment:
One more recommendation here that came out of my review session with @vpetrovykh:

Keep these two sentences as separate paragraphs since they are unrelated.

Also, add another one:

> In EdgeDB, arrays are always one-dimensional.

This isn't critical to the one-liner and can be relayed in the details instead.